recipe-server/client/control_new/components/recipes/Details.js

```diff
@@ -0,0 +1,48 @@
+import { Card, Tag } from 'antd';
+import { Map } from 'immutable';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+export default function Details({ recipe }) {
+  return (
+    <div>
+      <Card key="recipe-details" title="Recipe">
+        <dl className="details">
```

Comment:
I'm not sure. I think we should either add the data here as new UI, or migrate everything off.